resources/plugin.rb

```diff
@@ -17,10 +17,54 @@
 # limitations under the License.
 #
 
-actions :create, :remove
+include RundeckCookbook::Helpers
 
-default_action :create
+property :name, String,
+         name_attribute: true,
+         description: 'Name of the plugin to install'
```

Comment:
The description fields are only available in some of the last releases of Chef 13 and then Chef 14+ so be aware if you use these you're requiring fairly recent Chef releases and the metadata should be bumped to reflect that.